src/config/license.rs

```diff
@@ -211,7 +211,23 @@ impl TemplateParser {
     }
 }
 
-pub(crate) fn load_and_compile_template(path: &str) -> Result<Regex, LicenseError> {
+pub(crate) fn load_and_compile_template(
+    root_dir: &Option<std::path::PathBuf>,
+    path: &str,
+) -> Result<Regex, LicenseError> {
+    let mut path = std::path::PathBuf::from(path);
+    if path.is_relative() {
+        if let Some(root_dir) = root_dir {
+            // The tempate path needs to be resolved relative to the .rustfmt config file
+            // rather than whatever dir cargo fmt was run from.
+            let current = std::env::current_dir().expect("pwd");
+            std::env::set_current_dir(root_dir).expect("alter pwd");
+            if let Ok(abs_path) = path.canonicalize() {
```

Comment:
Hmm, https://github.com/rust-lang/rustfmt/pull/4131 suggests I use this instead:
```suggestion
            if let Ok(abs_path) = absolute_path(path) {
```

Reply:
Hmm but then I see dunce https://github.com/rust-lang/rustfmt/pull/4137 and then at some point it is removed... - trying to figure out why.

Reply:
shrugs. will go back to using canonicalize as that's used elsewhere in rustfmt. (Maybe canonicalize got better under windows?)